package.json

```diff
@@ -4,20 +4,24 @@
   "version": "0.0.1",
   "description": "Strimzi UI provides a way for managing Strimzi and Kafka clusters (+ other components) deployed by it using a graphical user interface.",
   "scripts": {
-    "clean": "rm -rf dist/ && rm -rf generated/",
-    "start": "webpack-dev-server --config utils/build/webpack.dev.js",
-    "build": "webpack --config utils/build/webpack.prod.js",
+    "clean": "rm -rf dist/ && rm -rf js/ && rm -rf generated/",
+    "start": "npm-run-all --parallel start:client start:server",
+    "start:client": "webpack-dev-server --config utils/build/webpack.dev.js",
+    "start:server": "npx nodemon --exec 'ts-node --project utils/build/tsconfig.server.json' server/main.ts",
+    "build": "webpack --config utils/build/webpack.client.prod.js && webpack --config utils/build/webpack.server.prod.js",
```

Comment:
Once typescript goes in there will be some conflicts here.  We can fix it and take this instead of what's on the typescript branch for scripts.

Reply:
Happy to discuss how/where we deliver the TS support. If it is easier, I am happy for this PR to implement yout TS reccomendations